test/modules/aws-rds-cluster-integration.ts

```diff
@@ -225,10 +225,10 @@ describe('DB Cluster Integration Testing', () => {
   it('starts a transaction', begin());
 
   itDocs(
```

Comment:
Can we put a query just before this where it tries to delete the delete-protected cluster, fails, and then runs a second query verifying that the record has been restored in the database?